docs/topics/cache.txt

```diff
@@ -900,7 +900,7 @@ Accessing the cache
 Basic usage
 -----------
 
-.. currentmodule:: django.core.caches
+.. currentmodule:: django.core.cache
```

Comment:
Good catch.
Can you open a separate PR for those cache-related changes so that this PR would contain only viewsource-related changes?

Reply:
https://github.com/django/django/pull/16885 good point - done :+1: